### DISCUSSION TITLE
Message failed: 450 Your trial domain reached its email quota limit.

### DISCUSSION THREAD

**Discussion by gabrielle-melo** [emailjs-com/help#141] [Q&A]
I'm using the free plan which has a limit of 200 emails per month, but when I was doing some email sending tests on the website when the limit reached 107 emails sent, emailjs started returning error 450, even though it returned that the limit was exceeded when trying to send again it increased the number of emails sent but did not send to the recipient, currently it says that I can send 100 emails in the month, but it continues to cause problems.

**Comment by xr0master** [comment]
Most likely, you get this error from your chosen mail provider. Since the EmailJS doesn't have this kind of error message.

**Comment by xr0master** [comment]
inactive
